### PR TITLE
Setup frontend project

### LINKED ISSUE #1: [xpt 1.0]Setup frontend project
- create react app, add linters,

### PR DESCRIPTION
**Pull request by dieum512**
In this repo I: 

- [x] Created a new React app
- [x] Installed the redux
- [x] Set up the linters configuration

Thank you for taking the time to review this project :wink:

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    {
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
```
Comment (binodbhusal) [inline public/manifest.json]: Please kindly remove the unnecessary files from the setup project
3CA18F96-E59B-43BB-8442-59B53D1208F6_1_201_a

**Review by binodbhusal — CHANGES_REQUESTED** [review]
Hi @dieum512 ,
Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

**Highlights**

- [x] No linter errors
- [x] Descriptive PR
- [x] App functionality working as expected. 👍

**Required Changes ♻️**
Check the comments under the review.
Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: public/manifest.json (hunk 1 above)
Comment (dieum512) [reply public/manifest.json]: Done :ok_hand:

**Review by binodbhusal — APPROVED** [review]
Hello @dieum512 😃 🥳 🎉 🎈
Great Job 😄 👏 You've done it. Your project is complete and approved! There is nothing else to say other than merge it 👍🏻 ✅
gif

To Highlight 💯

- [x] Good implementation of the required changes👍
- [x] Installed React and Redux and setup frontend project
- [x] well-organized structure✔ 👍
- [x] App work as expected

Congratulations! 🎉


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**